jq.pyx

```diff
@@ -51,6 +60,52 @@ cdef extern from "jq.h":
     void jq_get_error_cb(jq_state *, jq_err_cb *, void **)
 
 
+cdef object _jv_to_python(jv value):
+    """Unpack a jv value into a Python value"""
+    cdef jv_kind kind = jv_get_kind(value)
+    cdef int idx
+    cdef jv idx_key
+    cdef jv idx_value
```

Comment:
I don't think these are indices?

Reply:
I started them with `idx_` to indicate those are key/value at the `idx` index. As in "index'es key" and "index'es value". What would work for you better?

Reply:
Perhaps `property_name` and `property_value`?

Reply:
Sure. However, this way we would be assigning the return value of `jv_object_iter_key()` to `property_name`, which I think would be confusing. Like this:
```python
            property_name = jv_object_iter_key(value, idx)
```
So, for the start I renamed them to `property_key` and `property_value`. Tell me if you would prefer `property_name` regardless, and I'll change that.

Reply:
I don't think it makes too much difference. I think "name" matches the JSON spec, whereas "key" matches jq.

Reply:
Ah, now I see where `property_` comes from. Would it be OK that we store array elements in `property_value` as well?
Anyway, just thought I caught a mismatch here and tried to make it better. I think these are such minor details that it wouldn't matter in the end. I'll just put there whatever you'd like :)

Reply:
Ah, sorry, didn't notice you merged this already :D Thanks a lot :tada:!